.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "mne-dev",
+  // "image": "mcr.microsoft.com/devcontainers/base:debian-11",
+  "build": { "dockerfile": "Dockerfile" },
+  "containerEnv": {
+    "PYTHONNOUSERSITE": "true", // Make Python ignore the user's site-packages folder if it exists
+    "XDG_RUNTIME_DIR": "/home/vscode/.cache/xdgr"
```

Comment:
The base image I'm using uses the `vscode` user by default. There's been discussions on changing this to something more generic like `devcontainer`, but this still remains to be implemented:

https://github.com/devcontainers/images/issues/145

Changing it to a custom name like `mneuser` would require changes to the Dockerfile and potentially some of the features we're using; hence I'd like to refrain from doing that – there's no need to, you basically never deal with that username anyway outside of this very line here :)

I however changed the VNC password to `mne` in 54b159b

Reply:
I may have found a workaround but I need to test it a bit first. It may make it harder for me to get the dev container to work behind my corporate firewall, where I inject our self-signed SSL certs via a custom feature first. I want to make sure this doesn't become even more difficult or impossible if I change the default user name here.

Reply:
Ok but don't spend too much effort on it, it's a minor maintainer-facing improvement right (user will not notice)?

Reply:
It's basically irrelevant to users of the container, yes

Reply:
Username is now `mne-user`